tmail-backend/jmap/extensions-api/pom.xml

```diff
@@ -71,6 +71,15 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.ibm.icu</groupId>
+            <artifactId>icu4j</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.mnode.ical4j</groupId>
+            <artifactId>ical4j</artifactId>
+            <version>3.2.9</version>
```

Comment:
Count on me for forgetting to update the dependency, resulting in a delta, and non-previsible behaviour because of non allignement with Apache James.

We should:
 - Add ical4j in james project management and use it within the scope of Apache James projects
 - And then remove specific versions of Tmail so that we align with James BOM (bill of material)